lib/codebreaker/game.rb

```diff
@@ -0,0 +1,75 @@
+
+module Codebreaker
+  # //
```

Comment:
You can disable this rule in rubocop config file if you don't need to write descriptive comments.